src/ogc/wcs/WcsUrlBuilder.js

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright 2018 WorldWind Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * @exports WcsUrlBuilder
+ */
+define([
+        '../../error/ArgumentError',
+        '../../util/Logger'
+    ],
+    function (ArgumentError,
+              Logger) {
+        "use strict";
+
+        /**
+         * Constructs key value pair (KVP) WCS GetCoverage urls.
+         * @param wcsCoverage the WcsCoverage for which the urls should be generated
+         * @constructor
+         */
+        var WcsUrlBuilder = function (wcsCoverage) {
```

Comment:
Tight coupling between WcsCoverage and WcsUrlBuilder. What about providing the ID and service as construction arguments, since that's what's needed anyway?